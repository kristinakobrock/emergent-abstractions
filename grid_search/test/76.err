  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:02, 20.83it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:01, 25.01it/s] 18%|################7                                                                            | 9/50 [00:00<00:01, 23.24it/s] 24%|######################                                                                      | 12/50 [00:00<00:01, 22.65it/s] 30%|###########################6                                                                | 15/50 [00:00<00:01, 23.25it/s] 38%|##################################9                                                         | 19/50 [00:00<00:01, 24.59it/s] 44%|########################################4                                                   | 22/50 [00:00<00:01, 24.71it/s] 50%|##############################################                                              | 25/50 [00:01<00:01, 22.69it/s] 58%|#####################################################3                                      | 29/50 [00:01<00:00, 25.44it/s] 66%|############################################################7                               | 33/50 [00:01<00:00, 26.74it/s] 74%|####################################################################                        | 37/50 [00:01<00:00, 27.50it/s] 80%|#########################################################################6                  | 40/50 [00:01<00:00, 26.79it/s] 86%|###############################################################################1            | 43/50 [00:01<00:00, 27.45it/s] 92%|####################################################################################6       | 46/50 [00:01<00:00, 25.80it/s] 98%|##########################################################################################1 | 49/50 [00:01<00:00, 26.88it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.50it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 22.22it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 23.88it/s] 69%|################################################################3                            | 9/13 [00:00<00:00, 24.51it/s] 92%|####################################################################################9       | 12/13 [00:00<00:00, 24.75it/s]100%|############################################################################################| 13/13 [00:00<00:00, 24.76it/s]
