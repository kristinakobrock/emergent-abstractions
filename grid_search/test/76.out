# ["--attributes=5", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=32", "--learning_rate=0.001", "--hidden_size=128", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6668010950088501, "acc": 0.6210552453994751, "length": 3.2759225368499756, "mode": "train", "epoch": 1}
{"loss": 0.6648368239402771, "acc": 0.6246094107627869, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6570456624031067, "acc": 0.6306906342506409, "length": 2.450425863265991, "mode": "train", "epoch": 2}
{"loss": 0.6538953185081482, "acc": 0.6408325433731079, "length": 1.421875, "mode": "test", "epoch": 2}
{"loss": 0.6273478269577026, "acc": 0.6640124320983887, "length": 2.9401438236236572, "mode": "train", "epoch": 3}
{"loss": 0.6375454068183899, "acc": 0.6600279808044434, "length": 3.953125, "mode": "test", "epoch": 3}
{"loss": 0.5886366963386536, "acc": 0.6996359825134277, "length": 3.350379467010498, "mode": "train", "epoch": 4}
{"loss": 0.6121543049812317, "acc": 0.6826903820037842, "length": 3.44921875, "mode": "test", "epoch": 4}
{"loss": 0.5426470041275024, "acc": 0.7287895083427429, "length": 3.4119067192077637, "mode": "train", "epoch": 5}
{"loss": 0.594454824924469, "acc": 0.6905639171600342, "length": 3.703125, "mode": "test", "epoch": 5}
{"loss": 0.5208387970924377, "acc": 0.7418001890182495, "length": 3.753836154937744, "mode": "train", "epoch": 6}
{"loss": 0.6038776636123657, "acc": 0.7042175531387329, "length": 3.796875, "mode": "test", "epoch": 6}
{"loss": 0.5009667277336121, "acc": 0.757338285446167, "length": 3.714615821838379, "mode": "train", "epoch": 7}
{"loss": 0.5981064438819885, "acc": 0.7121825814247131, "length": 3.89453125, "mode": "test", "epoch": 7}
{"loss": 0.4835880398750305, "acc": 0.7682539224624634, "length": 3.7935121059417725, "mode": "train", "epoch": 8}
{"loss": 0.5355392694473267, "acc": 0.7336182594299316, "length": 3.890625, "mode": "test", "epoch": 8}
{"loss": 0.46073147654533386, "acc": 0.782950222492218, "length": 3.8014116287231445, "mode": "train", "epoch": 9}
{"loss": 0.5363858342170715, "acc": 0.7348998785018921, "length": 3.9140625, "mode": "test", "epoch": 9}
{"loss": 0.44699418544769287, "acc": 0.7887294292449951, "length": 3.874875545501709, "mode": "train", "epoch": 10}
{"loss": 0.5391896963119507, "acc": 0.7411742210388184, "length": 3.95703125, "mode": "test", "epoch": 10}
{"loss": 0.4304603040218353, "acc": 0.7927115559577942, "length": 3.913818359375, "mode": "train", "epoch": 11}
{"loss": 0.5448421835899353, "acc": 0.7407898306846619, "length": 3.98828125, "mode": "test", "epoch": 11}
{"loss": 0.42298033833503723, "acc": 0.8022644519805908, "length": 3.9490904808044434, "mode": "train", "epoch": 12}
{"loss": 0.5286864638328552, "acc": 0.7478514909744263, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.4142214357852936, "acc": 0.807774543762207, "length": 3.9686923027038574, "mode": "train", "epoch": 13}
{"loss": 0.5140941143035889, "acc": 0.7621093392372131, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.3991452753543854, "acc": 0.8174139857292175, "length": 3.971217632293701, "mode": "train", "epoch": 14}
{"loss": 0.5105736255645752, "acc": 0.7710937857627869, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.38568007946014404, "acc": 0.826225221157074, "length": 3.9799628257751465, "mode": "train", "epoch": 15}
{"loss": 0.4946879744529724, "acc": 0.7900390625, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.3671060800552368, "acc": 0.8370087742805481, "length": 3.985569477081299, "mode": "train", "epoch": 16}
{"loss": 0.46541181206703186, "acc": 0.7935547232627869, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.3541349768638611, "acc": 0.8438804149627686, "length": 3.9907748699188232, "mode": "train", "epoch": 17}
{"loss": 0.4826686680316925, "acc": 0.7923827767372131, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.3489459753036499, "acc": 0.8516198396682739, "length": 3.990262508392334, "mode": "train", "epoch": 18}
{"loss": 0.4664669334888458, "acc": 0.8130859732627869, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.34160301089286804, "acc": 0.8561236262321472, "length": 3.9892172813415527, "mode": "train", "epoch": 19}
{"loss": 0.4542621970176697, "acc": 0.8162841796875, "length": 3.99609375, "mode": "test", "epoch": 19}
{"loss": 0.3304341733455658, "acc": 0.8596798777580261, "length": 3.9931955337524414, "mode": "train", "epoch": 20}
{"loss": 0.44275766611099243, "acc": 0.8228515982627869, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.3210509717464447, "acc": 0.8668709397315979, "length": 3.9934988021850586, "mode": "train", "epoch": 21}
{"loss": 0.4296105206012726, "acc": 0.8306640982627869, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.30925559997558594, "acc": 0.871232807636261, "length": 3.9948506355285645, "mode": "train", "epoch": 22}
{"loss": 0.4093756079673767, "acc": 0.8365234732627869, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.3038659691810608, "acc": 0.8761267066001892, "length": 3.9962804317474365, "mode": "train", "epoch": 23}
{"loss": 0.42977771162986755, "acc": 0.8279296159744263, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.3017372190952301, "acc": 0.8785055875778198, "length": 3.996140480041504, "mode": "train", "epoch": 24}
{"loss": 0.4043036997318268, "acc": 0.8353515863418579, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.2895628809928894, "acc": 0.8852897882461548, "length": 3.9966278076171875, "mode": "train", "epoch": 25}
{"loss": 0.39494702219963074, "acc": 0.8453125357627869, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.28772810101509094, "acc": 0.8853493928909302, "length": 3.9971792697906494, "mode": "train", "epoch": 26}
{"loss": 0.42505550384521484, "acc": 0.8369140028953552, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.2830084562301636, "acc": 0.8883150219917297, "length": 3.9972846508026123, "mode": "train", "epoch": 27}
{"loss": 0.3869372308254242, "acc": 0.8472656607627869, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.2782856523990631, "acc": 0.8915448784828186, "length": 3.9993839263916016, "mode": "train", "epoch": 28}
{"loss": 0.379863977432251, "acc": 0.8568359017372131, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.27011752128601074, "acc": 0.8927299380302429, "length": 3.999627113342285, "mode": "train", "epoch": 29}
{"loss": 0.39093658328056335, "acc": 0.8457031846046448, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.2677537202835083, "acc": 0.8954806327819824, "length": 3.9986515045166016, "mode": "train", "epoch": 30}
{"loss": 0.3855031132698059, "acc": 0.8507813215255737, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.25969526171684265, "acc": 0.8984482884407043, "length": 3.9992926120758057, "mode": "train", "epoch": 31}
{"loss": 0.39242926239967346, "acc": 0.8560547232627869, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.2659303843975067, "acc": 0.8972737193107605, "length": 3.9996416568756104, "mode": "train", "epoch": 32}
{"loss": 0.3777795135974884, "acc": 0.8548827767372131, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.2605416178703308, "acc": 0.8965656757354736, "length": 3.9994966983795166, "mode": "train", "epoch": 33}
{"loss": 0.37879082560539246, "acc": 0.8597655892372131, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.254656583070755, "acc": 0.903535783290863, "length": 3.999603271484375, "mode": "train", "epoch": 34}
{"loss": 0.39280563592910767, "acc": 0.86328125, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.2524726092815399, "acc": 0.9040665626525879, "length": 3.9995341300964355, "mode": "train", "epoch": 35}
{"loss": 0.3888053894042969, "acc": 0.8585938215255737, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.2608201801776886, "acc": 0.9009733200073242, "length": 3.999096632003784, "mode": "train", "epoch": 36}
{"loss": 0.37625575065612793, "acc": 0.8607421517372131, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.2525811195373535, "acc": 0.9046173095703125, "length": 3.998487710952759, "mode": "train", "epoch": 37}
{"loss": 0.389684796333313, "acc": 0.8544921875, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.2465171068906784, "acc": 0.9067137837409973, "length": 3.9995627403259277, "mode": "train", "epoch": 38}
{"loss": 0.3711821734905243, "acc": 0.8671875, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.2456618696451187, "acc": 0.911033034324646, "length": 3.999821901321411, "mode": "train", "epoch": 39}
{"loss": 0.36483848094940186, "acc": 0.8675780892372131, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.24289710819721222, "acc": 0.9083799719810486, "length": 3.9998881816864014, "mode": "train", "epoch": 40}
{"loss": 0.3660983741283417, "acc": 0.8681641221046448, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.24317167699337006, "acc": 0.9096792340278625, "length": 3.9972872734069824, "mode": "train", "epoch": 41}
{"loss": 0.36876362562179565, "acc": 0.8636718392372131, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.24050664901733398, "acc": 0.9071996212005615, "length": 3.999936103820801, "mode": "train", "epoch": 42}
{"loss": 0.3814581334590912, "acc": 0.8660155534744263, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.2504503130912781, "acc": 0.907393753528595, "length": 3.99873948097229, "mode": "train", "epoch": 43}
{"loss": 0.37417352199554443, "acc": 0.8679686784744263, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.25195375084877014, "acc": 0.906785249710083, "length": 3.9998836517333984, "mode": "train", "epoch": 44}
{"loss": 0.3936176300048828, "acc": 0.8599609732627869, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.23950043320655823, "acc": 0.912779688835144, "length": 3.999385118484497, "mode": "train", "epoch": 45}
{"loss": 0.3711003363132477, "acc": 0.8642577528953552, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.23965567350387573, "acc": 0.9109756350517273, "length": 3.9998106956481934, "mode": "train", "epoch": 46}
{"loss": 0.35597074031829834, "acc": 0.8666015267372131, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.23324714601039886, "acc": 0.9148756265640259, "length": 3.9999890327453613, "mode": "train", "epoch": 47}
{"loss": 0.35936614871025085, "acc": 0.8705077767372131, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.22858686745166779, "acc": 0.9166448712348938, "length": 3.9999516010284424, "mode": "train", "epoch": 48}
{"loss": 0.353058397769928, "acc": 0.8718749284744263, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.22473792731761932, "acc": 0.9188305735588074, "length": 3.999988555908203, "mode": "train", "epoch": 49}
{"loss": 0.3767744302749634, "acc": 0.8689451813697815, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.21602408587932587, "acc": 0.9245362877845764, "length": 3.999464273452759, "mode": "train", "epoch": 50}
{"loss": 0.34937354922294617, "acc": 0.8683594465255737, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.2137051373720169, "acc": 0.9244383573532104, "length": 3.9999687671661377, "mode": "train", "epoch": 51}
{"loss": 0.32487615942955017, "acc": 0.8839844465255737, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.21079570055007935, "acc": 0.9259602427482605, "length": 3.9996633529663086, "mode": "train", "epoch": 52}
{"loss": 0.3269290328025818, "acc": 0.8849610686302185, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.2000352144241333, "acc": 0.9309316873550415, "length": 3.9999780654907227, "mode": "train", "epoch": 53}
{"loss": 0.3230869770050049, "acc": 0.8919921517372131, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.2091534286737442, "acc": 0.9279187917709351, "length": 3.999939441680908, "mode": "train", "epoch": 54}
{"loss": 0.33477193117141724, "acc": 0.88671875, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.20697082579135895, "acc": 0.9262689352035522, "length": 3.9999871253967285, "mode": "train", "epoch": 55}
{"loss": 0.34333258867263794, "acc": 0.87890625, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.21031229197978973, "acc": 0.9276261925697327, "length": 3.999981164932251, "mode": "train", "epoch": 56}
{"loss": 0.37468981742858887, "acc": 0.873046875, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.2076490968465805, "acc": 0.9269184470176697, "length": 3.999995470046997, "mode": "train", "epoch": 57}
{"loss": 0.31270769238471985, "acc": 0.8953126668930054, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.19243952631950378, "acc": 0.9343557953834534, "length": 3.9999871253967285, "mode": "train", "epoch": 58}
{"loss": 0.34467005729675293, "acc": 0.8814453482627869, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.1987254023551941, "acc": 0.9313458800315857, "length": 3.999999523162842, "mode": "train", "epoch": 59}
{"loss": 0.35000181198120117, "acc": 0.8792968392372131, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.19618426263332367, "acc": 0.9302459955215454, "length": 3.99857234954834, "mode": "train", "epoch": 60}
{"loss": 0.34902113676071167, "acc": 0.8777343034744263, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7590323090553284
